assets/src/pages/room/components/StreamPlayer/StatisticsLayer.tsx

```diff
@@ -0,0 +1,153 @@
+import { AudioStatistics, useDeveloperInfo, VideoStatistics } from "../../../../contexts/DeveloperInfoContext.tsx";
+import { useMemo } from "react";
+import { calculateAudioScore, calculateVideoScore } from "./rtcMosScore.ts";
+import { SIMULCAST_BANDWIDTH_LIMITS } from "../../bandwidth.tsx";
+
+
+const maxScoreHighLayer = calculateVideoScore({
+  codec: "",
+  bitrate: SIMULCAST_BANDWIDTH_LIMITS.get("h")! * 1024,
+  bufferDelay: 0,
+  roundTripTime: 0,
+  frameRate: 30,
+  expectedFrameRate: 30,
+  expectedWidth: 1280,
+  expectedHeight: 720
+});
+
+const maxScoreMediumLayer = calculateVideoScore({
+  codec: "",
+  bitrate: SIMULCAST_BANDWIDTH_LIMITS.get("m")! * 1024,
+  bufferDelay: 0,
+  roundTripTime: 0,
+  frameRate: 30,
+  expectedFrameRate: 30,
+  expectedWidth: 1280,
+  expectedHeight: 720
+});
+
+const maxScoreLowLayer = calculateVideoScore({
+  codec: "",
+  bitrate: SIMULCAST_BANDWIDTH_LIMITS.get("l")! * 1024,
+  bufferDelay: 0,
+  roundTripTime: 0,
+  frameRate: 30,
+  expectedFrameRate: 30,
+  expectedWidth: 1280,
+  expectedHeight: 720
+});
+
+const maxAudioScore = calculateAudioScore(
+  {
+    bitrate: 30_000,
+    bufferDelay: 0,
+    roundTripTime: 0,
+    packetLoss: 0,
+    fec: true,
+    dtx: false
+  });
+
+
+export type Props = { videoTrackId: string | null, audioTrackId: string | null }
+
+const numberFormatter = new Intl.NumberFormat("pl-PL", { minimumFractionDigits: 3 });
+
+export const StatisticsLayer = ({ videoTrackId, audioTrackId }: Props) => {
+
+  const { statistics } = useDeveloperInfo();
+
+  const videoRawStats = statistics.data[videoTrackId || ""];
+  const videoStats: VideoStatistics | undefined = videoRawStats?.type === "video" ? videoRawStats : undefined;
+
+  const videoScore = useMemo(() => {
+    if (!videoStats) return 0;
+
+    return calculateVideoScore(
+      {
+        codec: videoStats.codec,
+        bitrate: videoStats.bitrate,
+        bufferDelay: videoStats.bufferDelay,
+        roundTripTime: videoStats.roundTripTime,
+        frameRate: videoStats.frameRate,
+        expectedWidth: 1280,
+        expectedFrameRate: 24,
+        expectedHeight: 720
+      });
+  }, [videoStats]);
```

Comment:
How stable is the videoStats object? Maybe that useMemo is just introducing an overhead.